src/tunnel.cc

```diff
@@ -98,6 +98,9 @@ class TunnelStateData: public PeerSelectionInitiator
         return (server.conn != NULL && server.conn->getPeer() ? server.conn->getPeer()->host : request->url.host());
     };
 
+    /// start owning the given to-server connection; do not look for any other
```

Comment:
> We have differences of definition for "ownership" Ownership which make this description unclear about scope/timing.

Yeah. Soon we will be running out of standard, technical words we can use without arguments!

> I would prefer this were phrased in clearer terms. Something like this maybe:

```diff
- start owning the given to-server connection; do not look for any other
+ Finalize the given to-server connection for use. Retry or use of others is now prohibited.
```

The "finalize for use" part is too vague to be useful IMO. I have replaced "start owning" with "store" in hope to avoid the banned "ownership" word, but it makes the description less useful. Ownership is a valid technical term that is very useful in many contexts. We should not ban it, especially in places where it has very simple/reasonable/basic definition like "the object owner determines that object lifetime", even though the abuse of shared pointers makes such ownership claims less precise (they effectively talk about co-ownership rather than exclusive ownership).

Here is the current version:

```C++
/// store the given to-server connection; prohibit retries and do not look
/// for any other destinations
void commitToServer(const Comm::ConnectionPointer &);
```